src/utils.js

```diff
@@ -65,13 +65,19 @@ export const remapEvents = (contextAttrs) => {
   return result;
 };
 
-export const resetWebpackIcon = (Icon) => {
+export const resetWebpackIcon = async (Icon) => {
```

Comment:
Is this functionality directly related to webpack? If we want/need to run it even when not using webpack, then should it maybe be named differently, and if we _don't_, then should there be some sort of check to try to detect whether webpack is being used? Not sure what that would look like, just raising the question...

Reply:
@mikeu in a sense yes, it address a webpack bug, I think there is no harm in running this regardless of webpack (as I am aware) I think we can move on with this and then see if we get any issue / reports!